ssl/quic/quic_port.c

```diff
@@ -678,22 +845,13 @@ static void port_send_retry(QUIC_PORT *port,
                             QUIC_PKT_HDR *client_hdr)
 {
     BIO_MSG msg[1];
-    unsigned char buffer[512];
+    unsigned char buffer[512], *token_buf = NULL;
     WPACKET wpkt;
-    size_t written;
+    size_t written, token_buf_len;
     QUIC_PKT_HDR hdr;
```

Comment:
I'm afraid hdr = {0} here will be needed to make sanitizer checks happy.

Reply:
or you could change your marshaling of the QUIC_CONN_ID to only encode id_len bytes to the unused bytes don't get considered, and save 24 bytes of token space in the process